pennylane/pulse/parametrized_evolution.py

```diff
@@ -93,6 +93,8 @@ class ParametrizedEvolution(Operation):
             where :math:`t_i` are the additional times provided in ``t``.
             If ``True``, the *remaining* time evolution to :math:`t_f` is computed instead, returning
             :math:`\{U(t_0, t_f), U(t_1, t_f),\dots, U(t_{f-1}, t_f), U(t_f, t_f)\}`.
+        dense (bool): Whether the evolution should use dense matrices. Per default, this is decided by
+            the number of wires, i.e. ``dense = len(wires) < 3``.
```

Comment:
I guess strictly speaking `wires` is not "defined" at this level? maybe:
```suggestion
            the number of wires, i.e. ``dense = len(H.wires) < 3``.
```

Reply:
It is defined in the `super().__init__`

Reply:
But yes what you suggest is clearer 👍